x-pack/plugins/ml/server/shared_services/providers/results_service.ts

```diff
@@ -25,7 +25,14 @@ export function getResultsServiceProvider({
 }: SharedServicesChecks): ResultsServiceProvider {
   return {
     resultsServiceProvider(callAsCurrentUser: LegacyAPICaller, request: KibanaRequest) {
-      const hasMlCapabilities = getHasMlCapabilities(request);
+      //  Uptime is using this service in anomaly alert, kibana alerting doesn't provide request object
+      // So we are adding a dummy request for now
+      // TODO: Remove this once kibana alerting provides request object
```

Comment:
Assuming this TODO is left here by design, we are awaiting further implementation out of this PR's scope, right?

Reply:
Yes this workaround was suggested by ML team, siem is doing the same for this.